test/units/rewarder/HyperdriveCheckpointRewarder.t.sol

```diff
@@ -0,0 +1,170 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
Maybe name the file `MockHyperdriveCheckpointSubrewarder.t.sol` so it matched the contract name

Reply:
The file is named after the test contract (the second contract defined here). This is also the pattern used in the reentrancy tests FWIW